### PR TITLE
application.ymls prepared

### PR DESCRIPTION
**Pull request by hedgehog1833**
The application-ymls for preview and prod are prepared. Also the procfile was missing.

### CODE REVIEW

**Comment by hedgehog1833** [comment]
[OAuth2 Metal Detector - Butler](https://trello.com/c/HFHClh95/408-oauth2-metal-detector-butler)

File: app/src/main/resources/application-prod.yml [hunk 1]
```diff
@@ -8,7 +8,7 @@ server:
       enabled: false
 
 security:
-  issuer-uri: "http://localhost:9000"
+  issuer-uri: "http://auth-app:8080"
```
Comment (DanielW1987) [inline app/src/main/resources/application-prod.yml:11]: why not the public url?

Comment (hedgehog1833) [reply app/src/main/resources/application-prod.yml:11]: because you also did not do that for the butler 🤷‍♂️ 😉

Comment (DanielW1987) [reply app/src/main/resources/application-prod.yml:11]: Since we also use the public URL at Heroku, we should do that here too

**Review by DanielW1987 — APPROVED** [review]

**Pull request merged**